shows_app/src/app/components/feature/stepper/components/ShowsStep.tsx

```diff
@@ -0,0 +1,84 @@
+import React, { useContext } from "react";
+import { ShowContext } from "./ShowsContextProvider";
+import { Box, Flex, Image, Text } from "@chakra-ui/react";
+import { IShows } from "@/app/typings/shows";
+
+export const ShowsStep = () => {
+  const {
+    currentStep,
+    shows,
+    setCurrentStep,
+    firstRoundWinners,
+    setFirstRoundWinners,
+    secondRoundWinners,
+    setSecondRoundWinners,
+    setFinalWinner
+  } = useContext(ShowContext);
+
+  const handleSelection = (selectedShow: IShows) => {
+    if (currentStep < 4) {
+      setFirstRoundWinners([...firstRoundWinners, selectedShow]);
+    } else if (currentStep < 6) {
+      setSecondRoundWinners([...secondRoundWinners, selectedShow]);
+    } else {
+      setFinalWinner(selectedShow);
+    }
+    setCurrentStep(currentStep + 1);
+  };
+
+  function getShowsForCurrentStep() {
+    if (currentStep < 4) {
+      return Array.isArray(shows)
+        ? shows.slice(currentStep * 2, currentStep * 2 + 2)
```

Comment:
Is this console log here expected? What are the expected return values of this ternary?